src/Congo.WebApi/Startup.cs

```diff
@@ -42,13 +65,15 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                 app.UseDeveloperExceptionPage();
                 app.UseSwagger();
                 app.UseSwaggerUI(c => c.SwaggerEndpoint("/swagger/v1/swagger.json", "Congo.WebApi v1"));
-                app.UseCors(config => config.WithOrigins("*").AllowAnyMethod().AllowAnyHeader());
             }
 
             app.UseHttpsRedirection();
 
             app.UseRouting();
 
+            // https://docs.microsoft.com/en-us/aspnet/core/fundamentals/middleware/?view=aspnetcore-5.0#middleware-order-1
```

Comment:
Middleware ordering does indeed matter, but specifically the CORS one only has one ordering restriction: it must be before authorization/authentication, otherwise it doesn't matter. Still, I think its a nice resource to have available when messing with middlewares, so keep it, but perhaps put it at the top of the entire section?